src/dune_pkg/dune

```diff
@@ -7,10 +7,12 @@
   dune_engine
   dune_util
   dune_lang
+  dune_console
   opam_core
   opam_repository
   opam_format
   opam_state
-  opam_0install)
+  opam_0install
+  fmt)
```

Comment:
Where do we use fmt?

Reply:
Without this package it seems like the compiler doesn't know the definition of `Fmt.t` (`type 'a t = Format.formatter -> 'a -> unit`. If I remove this dependency I get an error:
```ocaml
File "src/dune_pkg/opam.ml", line 183, characters 18-32:
183 |       | Left l -> L.pp_rejection f l
                        ^^^^^^^^^^^^^^
Error: This expression has type L.rejection Fmt.t
       This is not a function; it cannot be applied.
```